app/src/main/java/dev/arkbuilders/arkmemo/ui/fragments/EditGraphicNotesFragment.kt

```diff
@@ -0,0 +1,103 @@
+package space.taran.arkmemo.ui.fragments
+
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import androidx.core.view.isVisible
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.viewModels
+import by.kirich1409.viewbindingdelegate.viewBinding
+import dagger.hilt.android.AndroidEntryPoint
+import space.taran.arkmemo.R
+import space.taran.arkmemo.ui.viewmodels.GraphicNotesViewModel
+import space.taran.arkmemo.databinding.FragmentEditNotesBinding
+import space.taran.arkmemo.models.Content
+import space.taran.arkmemo.models.GraphicNote
+import space.taran.arkmemo.ui.activities.MainActivity
+import space.taran.arkmemo.ui.viewmodels.NotesViewModel
+
+@AndroidEntryPoint
+class EditGraphicNotesFragment: Fragment(R.layout.fragment_edit_notes) {
+
+    private val activity by lazy {
+        requireActivity() as MainActivity
+    }
+
+    private val binding by viewBinding(FragmentEditNotesBinding::bind)
+
+    private val graphicNotesViewModel: GraphicNotesViewModel by viewModels()
+    private val notesViewModel: NotesViewModel by activityViewModels()
+
+    private var note = GraphicNote("", content = Content(""))
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        notesViewModel.init {}
+        if (arguments != null) {
+            requireArguments().getParcelable<GraphicNote>(GRAPHICAL_NOTE_KEY)?.let {
```

Comment:
This is a deprecated API.
![Screenshot 2023-11-10 at 21 31 46](https://github.com/ARK-Builders/ARK-Memo/assets/4281969/2a9a9846-adca-4846-9f3b-7fcf47e949a5)

Reply:
New API requires Android API leve 33, maybe we can have both for backward compatibility

Reply:
@ShubertMunthali what Android version is API level 33?

Reply:
Android 13

Reply:
Yeah, we want to support Android 10/11 at least. If we can support Android 9 without struggle it would be ideal.

@tuancoltech is it ok if we use this deprecated API for a while?

Reply:
@ShubertMunthali @kirillt Yes, using the deprecated API at the moment should not cause any issue, but we should start adapting to the new API because it might not work in Android's future release. Make it backward compatible like the latest code changes is a good choice.

Reply:
@tuancoltech could you create an issue for this, too? I'm not sure how to phrase it properly. Like "Support Android 14"?

Reply:
I have already added new API support plus backward compatibility. But later when we only support Andriod we may need to remove the backward code